homeassistant/components/anova_sous_vide/sensor.py

```diff
@@ -0,0 +1,168 @@
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        icon="mdi:clock-outline",
+        name="Cook Time",
+    ),
+    AnovaPrecisionCookerSensor.STATE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.STATE, name="State"
+    ),
+    AnovaPrecisionCookerSensor.MODE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.MODE, name="Mode"
+    ),
+    AnovaPrecisionCookerSensor.TARGET_TEMPERATURE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Target Temperature",
+    ),
+    AnovaPrecisionCookerSensor.COOK_TIME_REMAINING: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.COOK_TIME_REMAINING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        icon="mdi:clock-outline",
+        name="Cook Time Remaining",
+    ),
+    AnovaPrecisionCookerSensor.FIRMWARE_VERSION: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.FIRMWARE_VERSION,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        name="Firmware Version",
+    ),
+    AnovaPrecisionCookerSensor.HEATER_TEMPERATURE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.HEATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Heater Temperature",
+    ),
+    AnovaPrecisionCookerSensor.TRIAC_TEMPERATURE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TRIAC_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Triac Temperature",
+    ),
+    AnovaPrecisionCookerSensor.WATER_TEMPERATURE: SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.WATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Water Temperature",
+    ),
+}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Anova Sous Vide device."""
+    anova_wifi = hass.data[DOMAIN][entry.entry_id]
+    coordinator = AnovaCoordinator(hass, anova_wifi)
+    await coordinator.async_config_entry_first_refresh()
+    sensors = [
+        AnovaEntity(coordinator, description[1], description[0])
+        for description in SENSOR_DESCRIPTIONS.items()
```

Comment:
```suggestion
        AnovaEntity(coordinator, description, sensor)
        for sensor, description in SENSOR_DESCRIPTIONS.items()
```

Might be a bit more readable as for me